reporter/test/test_helpers.py

```diff
@@ -51,3 +51,11 @@ def test_generalize_sql(self):
 
         assert generalize_sql(sql) ==\
             "SELECT page_title FROM page WHERE page_namespace = X AND page_title COLLATE LATINN_GENERAL_CI LIKE X"
+
+    def test_get_method_from_query(self):
```

Comment:
Also would it be helpful to provide tests for some of the higher level logic i.e. the caching or pt_kill classes, by mocking kibana source and checking if processing of this data works correctly?